pkg/grafana/utils.go

```diff
@@ -27,3 +28,17 @@ func extractFolderUID(d DashboardWrapper) string {
 	}
 	return folderUid
 }
+
+func structToMap(s interface{}) (map[string]interface{}, error) {
+	jsonData, err := json.Marshal(s)
+	if err != nil {
+		return nil, err
+	}
+
+	var result map[string]interface{}
+	if err := json.Unmarshal(jsonData, &result); err != nil {
+		return nil, err
+	}
+
+	return result, nil
+}
```

Comment:
I thought https://github.com/mitchellh/mapstructure was in use somewhere in Grizzly. Seems not, but it does the same.

Reply:
I guess I'm biased by my early days in Go, when the community was generally more in favor (in comparison to other communities like JS) to avoid 3rd party dependencies just for few lines of code, so I didn't even consider it.

Reply:
However, after taking a look, I think it wouldn't be as simple as that.

These few lines of code are "re-using" the `json` tags defined in the client. But, https://github.com/mitchellh/mapstructure has its own tags (otherwise it would store them capitalized). 

So, I'd either need to add these flags in the client (I don't like it, tho I'm open for discussions 😄) or create the corresponding DTOs in Grizzly and map them before _decoding_, which is... even more lines of code and noise? 🤔 

Wdyt?